src/constants/mystery-encounters.ts

```diff
@@ -0,0 +1,50 @@
+/** Min - Max waves for mystery encounter in classic mode. */
+export const CLASSIC_MODE_MYSTERY_ENCOUNTER_WAVES: [min: number, max: number] = [10, 180];
+
+/** Min - Max waves for mystery encounter in challenge mode. */
+export const CHALLENGE_MODE_MYSTERY_ENCOUNTER_WAVES: [min: number, max: number] = [10, 180];
```

Comment:
We should freeze all arrays in constants

```suggestion
export const CHALLENGE_MODE_MYSTERY_ENCOUNTER_WAVES = Object.freeze<[min: number, max: number]>([10, 180]);
```